src/test/common/utils/interest.ts

```diff
@@ -0,0 +1,29 @@
+import { expect } from 'chai'
+import { getInterestDetails } from 'common/interest'
+import * as claimStoreServiceMock from '../../http-mocks/claim-store'
+import { Claim } from 'claims/models/claim'
+import { InterestType } from 'claim/form/models/interest'
+import { Moment } from 'moment'
+import { MomentFactory } from 'common/momentFactory'
+
+describe('getInterestDetails', () => {
+
+  it('should return undefined when interest has not been selected', async () => {
+    let claim: Claim = new Claim().deserialize(claimStoreServiceMock.sampleClaimObj)
+    claim.claimData.interest.type = InterestType.NO_INTEREST
+    expect(await getInterestDetails(claim)).to.be.eq(undefined)
+  })
+
+  it('should return expected interestDetails containing the needed values to be displayed', async () => {
+    const claim: Claim = new Claim().deserialize(claimStoreServiceMock.sampleClaimObj)
+    claimStoreServiceMock.mockCalculateInterestRate(0)
```

Comment:
an interest rate of 0 seems like an invalid test case? would you mind making it more realistic please?

Reply:
updated to 5%